src/inspector/view/qml/MuseScore/Inspector/text/TextSettings.qml

```diff
@@ -88,7 +88,7 @@ Column {
                 iconCode: modelData["iconRole"]
                 checked: root.model && !root.model.textScriptAlignment.isUndefined ? root.model.textScriptAlignment.value === modelData["typeRole"]
                                                                                    : false
-                onToggled: {
+                onClicked: {
```

Comment:
Could you please explain why this helps?

Reply:
I found that the `onToggled` signal is not emitted at all if the user clicks the radio button while it is already checked (because normally, the radio button would not be toggled in that case but just stay checked, so in general no need for a `toggled` signal).
But the `clicked` signal _is_ emitted, every time that the user clicks the button, also when it is already checked. That's exactly what we need in this case.